TCL4 Data Management/utm-tcl4-dmp-con1.yaml

```diff
@@ -242,7 +191,7 @@ definitions:
     type: array
     items:
       type: object
-      #required: []
+      required: [flightTestRangePolygonPoint_deg,flightTestRangePolygonPointElev_ft]
```

Comment:
So this is an array of arrays of arrays.  More concretely:  

`flightTestRangePolygonPoints` is an array with each element (`flightTestRangePolygonPoint_deg` and `flightTestRangePolygonPointElev_ft`) being arrays and `flightTestRangePolygonPoint_deg` being an array itself.

@dmulfinger please comment here.

This seems very error prone.  We must have an assumption that the elements of `flightTestRangePolygonPoint_deg` map to elements of `flightTestRangePolygonPointElev_ft` by array index?  

Any major reason we don't have this be an array of yoru `LatLonAlt` things?  Because someone won't like elevation being described as alt?

Needs discussion.